### PR TITLE
more types in fascades

### PR DESCRIPTION
**Pull request by kevmoo**
Also formatted the fascades code w/ dart_style

### CODE REVIEW

File: modules/facade/src/collection.dart [hunk 1]
```diff
@@ -5,69 +5,81 @@ export 'dart:core' show Map, List, Set;
...
-    this._iterator = map.keys.iterator;
-  }
-  bool moveNext() {
-    return this._iterator.moveNext();
-  }
   List get current {
-    return this._iterator.current != null ?
-      [this._iterator.current, this._map[this._iterator.current]] :
-      null;
+    return this._iterator.current != null
```
Comment (vicb) [inline modules/facade/src/collection.dart]: remove `this`

File: modules/facade/src/collection.dart [hunk 2]
```diff
@@ -5,69 +5,81 @@ export 'dart:core' show Map, List, Set;
...
-  static Iterable values(m) { return m.values; }
+  static get(Map m, k) => m[k];
+  static void set(Map m, k, v) {
+    m[k] = v;
+  }
+  static contains(Map m, k) => m.containsKey(k);
+  static forEach(Map m, fn(v, k)) {
+    m.forEach((k, v) => fn(v, k));
+  }
+  static int size(Map m) {
```
Comment (vicb) [inline modules/facade/src/collection.dart]: use `=>` where applicable for consistency ?

**Comment by vicb** [comment]
LGTM with a few comments. Could you please address them and rebase ?

File: modules/facade/src/collection.dart [hunk 3]
```diff
@@ -5,107 +5,119 @@ export 'dart:core' show Map, List, Set;
...
-  }
-  bool moveNext() {
-    return this._iterator.moveNext();
-  }
   List get current {
-    return this._iterator.current != null ?
-      [this._iterator.current, this._map[this._iterator.current]] :
-      null;
+    return _iterator.current != null
+        ? [_iterator.current, this._map[_iterator.current]]
```
Comment (vicb) [inline modules/facade/src/collection.dart]: I'll remove this this

Comment (kevmoo) [inline modules/facade/src/collection.dart]: Oops. Sorry

**Comment by vicb** [comment]
merged via 3ad62855958831e83eb3eb47202a67df6a24978f

File: modules/facade/src/collection.dart (hunk 3 above)
Comment (vicb) [inline modules/facade/src/collection.dart]: no pb, thanks for the PR

**Pull request closed without merging**